lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_methods.rb

```diff
@@ -99,6 +99,23 @@ def self.tag_create(category, options = {})
       end
     end
 
+    def self.tag_delete(category, entry, force_delete = false)
+      ar_method do
+        cat = Classification.find_by_name(category)
```

Comment:
Note, this `find_by_name` call is ok since the model is overriding the method: https://github.com/ManageIQ/manageiq/blob/master/app/models/classification.rb#L338

Reply:
Oh, thanks for pointing this out! I was trying to fix this but failed!